app/src/main/java/com/orange/ods/app/ui/MainScreen.kt

```diff
@@ -148,28 +150,29 @@ fun MainScreen(themeConfigurations: Set<OdsThemeConfigurationContract>, mainView
                         exit = fadeOut()
                     ) {
                         BottomBar(
-                            items = mainState.bottomBarItems,
-                            currentRoute = mainState.currentRoute!!,
+                            items = BottomBarItem.values(),
+                            currentRoute = mainState.navigationState.currentScreenRoute!!,
```

Comment:
This line is here since a long time but this PR is an opportunity to get rid of this `!!` and use `orEmpty()` instead.